src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -688,6 +707,11 @@ def test_run_app_bulk__dry_run(self, auth, c):
             self.assertTrue(expected_batch_run_keys == set(param_set.keys()))
         self.assertTrue(set(["wsid"]) == set(batch_params.keys()))
 
+        # expect shared_params to have been merged into respective param_sets
```

Comment:
Integration test for the change. With `dry_run=True` show that the output `param_set`s have had the `shared_params` merged in